web/.gitignore

```diff
@@ -0,0 +1,42 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
Should this be moved to the top-level .gitignore?

Reply:
I did some updates and merged it with the existing .gitignore